hardemu/keyboard.js

```diff
@@ -296,6 +296,13 @@ function Keyboard(Zeal, PIO) {
         return true;
     }
 
+    function str_press(str) {
```

Comment:
This seems to be related to your other feature about the editor, right? Can we remove it from this MR for now? :wink:

Reply:
#removed